pom.xml

```diff
@@ -395,6 +392,10 @@
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
           </exclusion>
+          <exclusion>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
```

Comment:
Exclude unnecessary dependencies to make the dependencies clean.
Use log4j-core as the implementation for slf4j.